core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -18,16 +18,40 @@
       GLOBALS.INTERACTION_SPECS = JSON.parse(
         '{{INTERACTION_SPECS|js_string}}');
     </script>
+
   {% endif %}
 
   {{dependencies_html}}
+  <style>
+   @media(max-width:768px) {
+     .navbar-collapse.collapse {
```

Comment:
navbar-collapse comes from third party library, I think you aren't allowed to change it.